contracts/callpaths/KnockoutPath.sol

```diff
@@ -82,6 +82,18 @@ contract KnockoutLiqPath is TradeMatcher, SettleLayer {
         // Ensure reserve flags are valid
         require(reserveFlags < 0x4, "RF");
 
+        if (base == address(0)) {
+            require(!isKnockoutCmd(code), "WB");
```

Comment:
we are saying `isKnockoutCmd` should be false here to pass this require, but when this call gets to `overCurve` it will fail with `Invalid command` as code wont match with any defined UserCmd

whats the intent of putting this require?

Reply:
We don't actually need the require, just copy pasting on my end. oops